test/Domain/LeanCode.CQRS.Tests/LoggingResponses/LoggingOperationResponsesTests.cs

```diff
@@ -0,0 +1,83 @@
+using LeanCode.CQRS.Execution;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Hosting.Internal;
+using Serilog;
+using Serilog.Events;
+using Xunit;
+
+namespace LeanCode.CQRS.Tests
+{
+    public class LoggingOperationResponsesTests : BaseCQRSTests
+    {
+        const string OperationResponse = "Success!";
+
+        [Fact]
+        public async Task Correctly_log_operation_response_and_run_next()
+        {
+            Prepare();
+
+            var appCtx = new AppContext();
+            var fakeSink = new FakeSink();
+
+            var logger = new LoggerConfiguration().MinimumLevel
+                .Is(LogEventLevel.Verbose)
+                .WriteTo.Sink(fakeSink, LogEventLevel.Verbose)
+                .CreateLogger();
+
+            var element = new ResponseLoggerElement<AppContext, object, object>(logger);
+
+            var result = await element.ExecuteAsync(appCtx, OperationResponse, (ctx, obj) => Task.FromResult<object>(obj));
+
+            Assert.Single(fakeSink.Messages);
+            Assert.Equal(OperationResponse, result);
+            Assert.Contains(result.ToString(), fakeSink.Messages[0]);
+        }
+
+        [Fact]
+        public async Task Correctly_log_operation_response_and_run_next_on_dev_env()
+        {
+            Prepare();
+
+            var appCtx = new AppContext();
+            var fakeSink = new FakeSink();
+
+            IHostEnvironment env = new HostingEnvironment { EnvironmentName = Environments.Development };
+
+            var logger = new LoggerConfiguration().MinimumLevel
+                .Is(LogEventLevel.Verbose)
+                .WriteTo.Sink(fakeSink, LogEventLevel.Verbose)
+                .CreateLogger();
+
+            var element = new DevResponseLoggerElement<AppContext, object, object>(env, logger);
+
+            var result = await element.ExecuteAsync(appCtx, OperationResponse, (ctx, obj) => Task.FromResult<object>(obj));
+
+            Assert.Single(fakeSink.Messages);
+            Assert.Equal(OperationResponse, result);
+            Assert.Contains(result.ToString(), fakeSink.Messages[0]);
+        }
+
+        [Fact]
+        public async Task Does_not_log_operation_response_and_run_next_on_prod_env()
```

Comment:
These are two tests - one that checks if the message is logged, one that checks if `next` is called.